pragma/core/entry.py

```diff
@@ -285,20 +291,28 @@ def from_dict(entry_dict: Dict[str, str]) -> "FutureEntry":
             autoscale_volume=False,
         )
 
-    @staticmethod
```

Comment:
why can't it stay a staticmethod ?

Reply:
@EvolveArt Because the abstract base class has it as a `classmethod`. `staticmethod` and `classmethod` cannot be interchanged without triggering a function signature conflict since `staticmethod` does not accept a `cls` parameter. Happy to change this back to a `staticmethod` and disable this in the `pyright` config!

Reply:
it's mean to be called like `SpotEntry.serialize_entries` so I think it should stay a static method, wdyt ?

Reply:
Nevermind, reading on the subject I think that does make sense

Reply:
@EvolveArt that sounds good to me. Should we also change the abstract base class' (`Entry`) function signature too, then?

Reply:
Given the fact that `self` is not being used, it does make more sense no?

```python
def serialize_entries(self, entries: List[Entry]):
        serialized_entries = [
            entry.serialize() for entry in entries if issubclass(entry, Entry)
        ]
        return list(filter(lambda item: item is not None, serialized_entries))
```

Reply:
Yes, agreed!

Reply:
change done!